Framework/Geometry/inc/MantidGeometry/Instrument.h

```diff
@@ -6,25 +6,22 @@
 #include "MantidGeometry/IDetector.h"
 #include "MantidGeometry/Instrument/CompAssembly.h"
 #include "MantidGeometry/Instrument/ObjComponent.h"
-
 #include "MantidKernel/DateAndTime.h"
+#include "MantidBeamline/Beamline.h"
```

Comment:
Considered that, and sat of the fence for a while - eventually decided that ownership and copying semantics issues were complex enough that I wanted to shield from the client. will define further below.